src/diy/smol-slimes/smol-hardware.md

```diff
@@ -0,0 +1,205 @@
+
+ - BMI270
+ - ICM-42688-P
+ - ICM-42688-V
+ - ICM-45686
+ - ISM330BX
+ - ISM330DHCX
+ - ISM330DLC
+ - LSM6DS3
+ - LSM6DS3TR-C
+ - LSM6DSL
+ - LSM6DSM
+ - LSM6DSO
+ - LSM6DSR
+ - LSM6DSV
+ - LSM6DSV16B
+
+### 🧲 Magnetometers
+ - AK09940
+ - <div class="tooltip-text-container">BMM150
+    <span class="tooltip-text">Sensor driver has not been tested.</span>
+   </div>
+ - <div class="tooltip-text-container">BMM350
+    <span class="tooltip-text">Sensor driver has not been tested.</span>
+   </div>
+ - IIS2MDC
+ - IST8306
+ - IST8308
+ - LIS2MDL
+ - <div class="tooltip-text-container">LIS3MDL
+    <span class="tooltip-text">Sensor driver has not been tested.</span>
+   </div>
+ - MMC5983MA
+
+### 🟩 Sensor Modules with IMU and Magnetometer
+**Sensor Compatibility Notice**
+- **Firmware Limitations**: Current firmware does not support BMI160/BMI270 sensor modules.  
+- **Hardware Restrictions**: MPU-6050-based trackers lack required features for full functionality.  
+- **Community Solutions**: Third-party forks with experimental support exist but are not officially maintained.  
+- **Check Compatibility**: Verify your hardware version [here](#sensor-compatibility-chart) before proceeding.
+
+##### Meia IMU + Magnetometer Modules
+Meia, a member of the SlimeVR Discord, produces and sells IMUs with an onboard magnetometer suitable for stacked builds.
+
+<table>
+    <tr>
+        <th>IMU + Magnetometer</th>
+        <th>Product Page</th>
+    </tr>
+    <tr>
+        <td><a href="../diy/imu-comparison.md#ICM-45686">ICM-45686</a> + IST8306</td>
+        <td><a href="https://store.kouno.xyz/products/icm-45686-ist8306-module">store.kouno.xyz</a></td>
+    </tr>
+    <tr>
+        <td>LSM6DSR + IST8306</td>
+        <td><a href="https://store.kouno.xyz/products/lsm6dsr-ist8306-module">store.kouno.xyz</a></td>
+    </tr>
+</table>
+
+### 🖲️ Buttons
+Push buttons and momentary switches are utilized to control the tracker. The functions of this button—Reset, Calibration, Pairing, Deep Sleep, and entering DFU Mode—depend on the number of press combinations. A tracker can be equipped with either a reset button, a user-specified (SW0) button, or both.
+
+The reset button is designed to support all functionalities. If an user-specified button (SW0) is defined, it will be utilized instead.
+
+If a button is unavailable, tweezers can be used to short the pins for the initial tracker setup.
+
+### 🕹️ Switches
+A slide switch can be used to physically disconnect a battery. Some boards have a high standby power draw and will require a switch.
+
+If a switch is not utilized, a tracker can enter Deep Sleep mode by pressing and holding down the user-specified button (SW0).
+
+### 🔋 Batteries
+Batteries are rated by capacity (mAh) and have a maximum safe charging rate (C). Charging at lower rates (e.g., around 0.5C) is recommended to reduce battery stress and extend lifespan.
```

Comment:
"Safe battery charging rates (C) are correlated to their rated capacity (mAh). A 100mAh battery charging at 100mA is 1C, and a 200mAh battery charging at 100mA is 0.5C. Charging at lower rates near 0.5C is recommended to reduce battery stress and extend lifespan."